sympy/utilities/lambdify.py

```diff
@@ -184,8 +189,8 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     standard library ``math``, or ``mpmath`` functions otherwise. To change
     this behavior, the "modules" argument can be used. It accepts:
 
-     - the strings "math", "mpmath", "numpy", "numexpr", "scipy", "sympy",
-       "tensorflow"
+     - the strings "math", "mpmath", "numpy", "autogra.numpy", "numexpr",
+     "scipy", "sympy", "tensorflow"
```

Comment:
I think the Sphinx error is because this line needs to be indented. 
```suggestion
       "scipy", "sympy", "tensorflow"
```